workspace/welcome-to-xod/206-fade/patch.xodp

```diff
@@ -1,15 +1,15 @@
 {
   "comments": [
     {
-      "content": "# Fade\n\nHere is a program with the LED that turns on smoothly while the button pressed.\n\nTo make a smooth transition from 0 to 1 and vice versa we use the `fade` node from the XOD standard library. It transforms its output value toward the target value with the rate specified.\n\nIn some cases this node helps to reduce bouncing or power bursts. For example, when you rotate a motor without smooth acceleration, it can cause a voltage drop and whole device reboot.\n\n## Instructions\n\n1. Replace `led` with `servo`.\n\n2. Set 2 to the `RATE` value to get a faster transition.\n\n3. Upload the program.\n\n4. Press and hold the button a few times.\n\nRemove the `fade` node and take a look at the board. When the motor starts moving you can see board LEDs are dimming; the board even can reboot.",
+      "content": "# Fade\n\nHere is a program with the LED that turns on smoothly while the button pressed.\n\nTo make a smooth transition from 0 to 1 and vice versa we use the `fade` node from the XOD standard library. It transforms its output value toward the target value with the rate specified.\n\nIn some cases this node helps to reduce bouncing or power bursts. For example, when you rotate a motor without smooth acceleration, it can cause a voltage drop and whole device reboot.\n\n## Exercise\n\nTest the `fade` node with `servo`.\n\n1. Add a `servo` next to `led` and link it with `fade` too.\n\n2. Bind 2.0 to the `RATE` pin to get a faster transition.\n\n3. Upload the program.\n\nPress and hold the button a few times. Observe how `fade` smoothens the transition and changes the direction if the re-targeted in the middle of the process.\n\nRemove the `fade` node, link to `button` directly, and take a look at the board. When the motor starts moving you can see board LEDs are dimming; the board even can reboot.",
```

Comment:
```suggestion
      "content": "# Fade\n\nHere is a program with the LED that turns on smoothly while the button is pressed.\n\nTo make a smooth transition from 0 to 1 and vice versa we use the `fade` node from the XOD standard library. It transforms its output value toward the target value with the rate specified.\n\nIn some cases this node helps to reduce bouncing or power bursts. For example, when you rotate a motor without smooth acceleration, it can cause a voltage drop and whole device reboot.\n\n## Exercise\n\nTest the `fade` node with `servo`.\n\n1. Add a `servo` next to `led` and link it with `fade` too.\n\n2. Bind 2.0 to the `RATE` pin to get a faster transition.\n\n3. Upload the program.\n\nPress and hold the button a few times. Observe how `fade` smoothens the transition and changes the direction if the re-targeted in the middle of the process.\n\nRemove the `fade` node, link to `button` directly, and take a look at the board. When the motor starts moving you can see board LEDs are dimming; the board even can reboot.",
```

"while the button pressed" -> "while the button is pressed"